app/views/TaxCodeDetailsPreviousYearsView.scala.html

```diff
@@ -77,8 +79,13 @@ <h2 class="govuk-!-margin-top-5 govuk-heading-m" id="taxCodeList@{listIdx.toStri
                 }
 
             </div>
+            <div class="print-this">
+                <a class="print-this__link" href="javascript:window.print()">Print this page </a>
```

Comment:
This probs needs to be a message key, not sure if it will translate to welsh otherwise